examples/cra-kitchen-sink/.storybook/config.js

```diff
@@ -1,9 +1,12 @@
 import { configure, setAddon } from '@storybook/react';
 import { setOptions } from '@storybook/addon-options';
+import { setViewports } from '@storybook/addon-viewport';
 
 // deprecated usage of infoAddon
 import infoAddon from '@storybook/addon-info';
 
+import viewports from './viewports.json';
```

Comment:
I think that you need to move the example to the `official-storybook`. @Hypnosphi  ?

Reply:
Yes, that would be right

Reply:
@igor-dv @Hypnosphi I moved it to `official-storybook`